test/native_objc_test/automated_ref_count_test.dart

```diff
@@ -42,12 +42,20 @@ void main() {
     }
 
     verifyRefCountsInner(Pointer<Int32> counter) {
-      final obj1 = ArcTestObject.alloc(lib).initWithCounter_(counter);
+      final obj1 = ArcTestObject.new1(lib);
```

Comment:
Is it possible to structure the tests so that it is more clear what scenario you are testing?